xml/System.Buffers/StandardFormat.xml

```diff
@@ -90,9 +103,15 @@
       </Parameters>
       <Docs>
         <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Indicates whether the current instances is equal to another <see cref="T:System.Buffers.StandardFormat" /> instance.</summary>
```

Comment:
should the two overloads have similar summaries?
![image](https://user-images.githubusercontent.com/12971179/59131289-d82b7500-8926-11e9-8c87-c2497c2e4822.png)